### ISSUE TITLE
Allow multiple form submissions

### ISSUE THREAD

**Issue by vasikgreif** [formvalidation/formvalidation#1422]
Is there a setting to allow multiple form submissions?

**Comment by nghuuphuoc** [comment]
I'm afraid that it's not possible at the moment.
Can you tell me the scenario that might need it?

**Comment by vasikgreif** [comment]
I have a search form, that I'm validating and submitting via Ajax. The user submits the first search, he gets some results, and needs to adjust the search options to get a bit different results.

So, on the Ajax success I need to enable resubmitting the form again...

**Comment by nghuuphuoc** [comment]
Maybe the [Using Ajax to submit the form](http://formvalidation.io/examples/ajax-submit/) example and the [defaultSubmit()](http://formvalidation.io/api/#default-submit) method help you.

**Comment by vasikgreif** [comment]
No, I need to first validate the form, submit it and than enable resubmit in Ajax callback - I don't see how the defaultSubmit() method should help with that.
